practical-penguins/setup.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+
+import pkg_resources
+import setuptools
+
+with pathlib.Path('requirements.txt').open() as requirements_txt:
+    install_requires = [
+        str(requirement)
+        for requirement
+        in pkg_resources.parse_requirements(requirements_txt)
+    ]
+
```

Comment:
This is very clever and nice use of pathlib

Reply:
Indeed, I'm quite a fan of this pathlib usage for gathering the requirements. My only comment would be to *slightly* condense it, as something like:

```py

    install_requires = [
        str(requirement) for requirement 
        in pkg_resources.parse_requirements(requirements_txt)
    ]
```

But that's a bit of a nit, and somewhat comes down to personal preference. :-)